appsec/src/helper/subscriber/waf.cpp

```diff
@@ -527,7 +529,7 @@ void instance::listener::submit_metrics(
         tags.add("waf_timeout", "true");
     }
     if (waf_run_error_) {
-        tags.add("waf_run_error", "true");
+        tags.add("waf_error", "true");
```

Comment:
the doc says "waf_error" sould contain the numerical error returned by ddwaf_run, or -127 if the error happened in the binding. Which is why it's not enough to store a count earlier.

Reply:
There are two different tags for `waf_error` if you check https://docs.google.com/document/d/1D4hkC0jwwUyeo0hEQgyKP54kM1LZU98GL8MaP60tQrA/edit?tab=t.0
This one is the one for `appsec.waf.requests` and therefore it's ok.

The one you're refering to(`appsec.waf.error`) is not implemented on this PR

Reply:
You're right, but below you added `appsec.rasp.error` without including the code as a tag

Reply:
Yes because that tag is not part of the SLO